saskatoon/sitebase/templates/app/detail_views/property/harvest_row.html

```diff
@@ -1,7 +1,37 @@
 {% load i18n %}
 {% load static %}
 <tr>
-    <td>{{ harvest.pick_leader__person__first_name }}</td>
+    <td>
+        <a href='/harvest/{{harvest.id}}'>#{{ harvest.id }} Harvest by {{ harvest.pick_leader.name }}</a>
+        {% if perms.harvest.delete_harvest %}
+            &nbsp;
+            <small>
+                <a href="/admin/harvest/harvest/{{harvest.id}}/delete"
```

Comment:
@mina-andajos could you refactor reversed urls for this one as you did in #276 ?